src/datatype.c

```diff
@@ -296,165 +296,183 @@ static int references_name(jl_value_t *p, jl_typename_t *name) JL_NOTSAFEPOINT
     return 0;
 }
 
+static void throw_ovf(int should_malloc, void *desc, jl_datatype_t* st, int offset)
+{
+    if (should_malloc)
```

Comment:
`should_free`?

Reply:
I suppose it could. It's currently just carrying the same name along.

Reply:
Do you care?

Reply:
Not so important I guess.